repo/blob/s3/s3_options.go

```diff
@@ -19,9 +19,16 @@ type Options struct {
 	DoNotVerifyTLS bool   `json:"doNotVerifyTLS,omitempty"`
 	RootCA         []byte `json:"rootCA,omitempty"`
 
-	AccessKeyID     string `json:"accessKeyID"`
-	SecretAccessKey string `json:"secretAccessKey" kopia:"sensitive"`
-	SessionToken    string `json:"sessionToken"    kopia:"sensitive"`
+	AccessKeyID     string            `json:"accessKeyID"`
+	SecretAccessKey string            `json:"secretAccessKey" kopia:"sensitive"`
+	SessionToken    string            `json:"sessionToken" kopia:"sensitive"`
+	Tags            map[string]string `json:"tags" kopia:"sensitive"`
+	RoleARN         string            `json:"roleARN"`
+	SessionName     string            `json:"sessionName"`
+	RoleDuration    time.Duration     `json:"duration"`
```

Comment:
This is probably OK, if it is not meant to be read or modified by humans.
The approach in #4194 would allow human-friendly marshaling.

Reply:
yea, we can pull in something like #4194 since it does make it more readable in logs and whatnot. I was mostly trying to get a minimal working set of code using just minio for the moment 😄